server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -288,14 +289,13 @@ public void testHashJoin_TableOrderInLogicalAndExecutionPlan() {
         rowCountByTable.put(TEST_DOC_LOCATIONS_TABLE_IDENT, new Stats(10, 0, Map.of()));
         e.updateTableStats(rowCountByTable);
 
-        LogicalPlan operator = createLogicalPlan(mss);
-        assertThat(operator).isExactlyInstanceOf(HashJoin.class);
-        assertThat(((HashJoin) operator).rhs().getRelationNames())
-            .as("Smaller table must be on the right-hand-side")
```

Comment:
Would be cool if we can preserve things like this `.as` in the new assertion. Afaik it's still accurate and useful information that clarifies the test intention?

Same in another test case further below

Reply:
ok, will do.